src/helm/proxy/clients/huggingface_tokenizer.py

```diff
@@ -1,84 +1,67 @@
 import os
-from typing import Any, Dict, Optional
+from typing import Dict, Optional
+from threading import Lock
 
 from transformers import AutoTokenizer
 
 from helm.common.hierarchical_logger import htrack_block, hlog
 
-from helm.proxy.clients.huggingface_model_registry import (
-    get_huggingface_model_config,
-    HuggingFaceHubModelConfig,
-    HuggingFaceLocalModelConfig,
-)
 
+# TODO: Rename HuggingFaceTokenizer to disambiguate Hugging Face AutoTokenizers and HELM Tokenizers.
+# TODO: Rename HuggingFaceTokenizers to a Factory.
```

Comment:
Should we not do the renaming in this PR?

Reply:
This touches a lot of files, so I'd prefer to do this later.